src/lib/error.ts

```diff
@@ -6,6 +6,11 @@ export enum ErrorCode {
     AuthenticationRequired = 'You are not authenticated!',
     UnsupportedAuthorization = 'Unsupported authorization type!',
     MissingRefreshToken = 'Refresh token is required parameter! Please send it in body or query as refreshToken.',
+    UserByResetTokenNotImplemented = "Function 'getUserByResetPasswordToken' is not implemented!",
+    UpdatePasswordNotImplemented = "Function 'updatePassword' is not implemented!",
+    ResetPasswordTokenRequired = 'Token is required field, please enter the token!',
```

Comment:
Entering is a process for the end aplication, not the backend. Make it clear that the token is not available in the request, as you have for the UserByResetTokenNotImplemented.

Reply:
changed: `Reset password token is required parameter! Please send it in body or query as token.`